src/movement/control_system.cpp

```diff
@@ -42,30 +42,33 @@ ControlSystem::ControlSystem(ros::NodeHandle *_nh, ros::Publisher *_pub)
     // Thruster settings. Don't have nodehandle ref so must use ros::param::get
     XmlRpc::XmlRpcValue thruster_settings;
 
-    if(!ros::param::get("thrusters", thruster_settings))
+    if(!ros::param::get("thruster", thruster_settings))
     {
         ROS_FATAL("thruster params failed to load");
         exit(1);
     }
 
+    ROS_INFO_STREAM(thruster_settings.size());
```

Comment:
Perhaps something a little more obvious on the output like: `"Loaded " << thruster_settings.size() << " thrusters." This would just make things clearer in the output to someone not familiar with the control system.